microsoft-365/security/office-365-security/view-and-release-quarantined-messages-from-shared-mailboxes.md

```diff
@@ -47,6 +47,8 @@ Regardless of the method, users can avoid confusion by including the **Recipient
 
 - Currently, the **Block sender** button is not available in the **Details** flyout for quarantined messages that were sent to the shared mailbox.
 
+- For access that are granted to users through the use of security groups, in order for the quarantine operation to be performant we recommend not adding permissions through more than two levels of nesting on security groups. for example, an Admin wants to give User "a" access to shared mailbox "XYZ". If the Admin has security groups "DEF@contoso.com", "HIJ@contoso.com", "KLM@contoso.com" and they all have stand alone access to shared mailbox "XYZ". we recommend assigning User "a" access directly to security group "DEF@contoso.com," or directly to security group "HIJ@contoso.com," or directly to security group "KLM@contoso.com". instead of nesting "DEF@contoso.com" to "HIJ@contoso.com" to "KLM@contoso.com" and later assigning User "a" access to "DEF@contoso.com" security group. 
```

Comment:
```suggestion
- For access that is granted to users through the use of security groups in order for the quarantine operation to be performed, we recommend not adding permissions through more than two levels of nesting on security groups. For example, an admin wants to give user **a** access to shared mailbox **XYZ**. If the admin has security groups _DEF@contoso.com_, _HIJ@contoso.com_, _KLM@contoso.com_, and they all have stand-alone access to shared mailbox **XYZ**, we recommend assigning User **a** access directly to security group _DEF@contoso.com_, or directly to security group _HIJ@contoso.com_, or directly to security group _KLM@contoso.com_, instead of nesting _DEF@contoso.com_ to _HIJ@contoso.com_ to _KLM@contoso.com_ and later assigning User **a** access to the _DEF@contoso.com_ security group. 
```